### PR TITLE
Fetching data

### PR DESCRIPTION
**Pull request by yousef-najeh**

### CODE REVIEW

File: thirdstage-react/src/Containers/Contact/contact.jsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 /* eslint-disable no-unused-vars */
-import { Field, Form, Formik} from "formik"
+
+import {Form, Formik} from "formik"
 import commentSchema from "../../schemas/BaisicSchema/"
 import Input from "../../component/InputField/input"
+import inputsField from "../../Data/ContactSectionData"
 import "./contactStyle.scss"
-import inputsField from "../../Data/ContactData"
```
Comment (amonamoqadi) [inline thirdstage-react/src/Containers/Contact/contact.jsx]: add semicolon

File: thirdstage-react/src/Containers/Contact/contact.jsx [hunk 2]
```diff
@@ -12,7 +13,7 @@ function Contact(){
             <p className = "contact__header">Contact</p>
             <p className = "contact__greeting">lets get in touch when you finish</p>
             <Formik 
-                initialValues = {{
+                initialValues={{
                 name:'',
                 email: '',
                 subject: '',
```
Comment (amonamoqadi) [inline thirdstage-react/src/Containers/Contact/contact.jsx]: not formatted well

File: thirdstage-react/src/Containers/Contact/contact.jsx [hunk 3]
```diff
@@ -40,6 +41,7 @@ function Contact(){
                             className = "contact__form__submit">
                         Submit
                     </button>
+
```
Comment (amonamoqadi) [inline thirdstage-react/src/Containers/Contact/contact.jsx]: remove

File: thirdstage-react/src/Containers/Contact/contactStyle.scss [hunk 4]
```diff
@@ -21,9 +21,9 @@
                 width:200px;
                 padding: 10px 5px;
                 background-color: black;
-                color:white;
+
```
Comment (amonamoqadi) [inline thirdstage-react/src/Containers/Contact/contactStyle.scss]: remove extra line

File: thirdstage-react/src/Router/router.jsx [hunk 5]
```diff
@@ -4,20 +4,24 @@ import {
...
     createRoutesFromElements(
-        <Route path="/" element = {<RootLayout/>}>
+        <Route path = "/" element = {<RootLayout/>}>
                 <Route index element = {<MainPage/>}/>
-                <Route path="Projects" element = {<Projects/>}/>
-                <Route path="About" element = {<About/>}/>
-                <Route path="Contact" element = {<Contact/>}/>
+                <Route path = "Projects" element = {<Projects/>}/>
+                <Route path = "About" element = {<About cardsNumber = {1}/>}/>
+                <Route path = "Contact" element = {<Contact/>}/>
```
Comment (amonamoqadi) [inline thirdstage-react/src/Router/router.jsx]: no need to add space before and after =

Comment (yousef-najeh) [reply thirdstage-react/src/Router/router.jsx]: so should i add after or before =?

Comment (yousef-najeh) [reply thirdstage-react/src/Router/router.jsx]: @amonamoqadi

**Pull request merged**